### PR TITLE
Add Form Allocation of Payments

### LINKED ISSUE #136: Add support to payment allocation form
Greetings, this is how the payment allocation form is displayed in the customer ZK for ADempiere.
Screenshot_20191119_145805

This form must be migrated to the new Vue para ADempiere client.

### PR DESCRIPTION
**Pull request by elsiosanchez**
## Feature
Proposal for the payment assignment form

#### Screenshot or Gif
Behavior when there is a difference between payment and invoice
success

error

#### Other relevant information
- Your OS: Debian 9
- Web Browser: Chrome
- Node.js version: 18.0

#### Issues
fixes #136

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -33,5 +33,8 @@
   },
   "priceChecking": {
     "endpoint": "/form/addons/point-of-sales"
+  },
+  "allocationPayments": {
```
Comment (yamelsenih) [inline config/default.json]: change by "paymentAllocation"

File: src/api/ADempiere/form/v-allocation.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+// ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
```
Comment (yamelsenih) [inline src/api/ADempiere/form/v-allocation.js]: Change name by paymentAllocation

File: src/components/ADempiere/Form/allocationPayments/Invoices/index.vue [hunk 3]
```diff
@@ -0,0 +1,61 @@
+<!--
+ ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/allocationPayments/Invoices/index.vue]: Change folder by paymentAllocation

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/Form/allocationPayments/Summary/index.vue [hunk 4]
```diff
@@ -41,6 +41,9 @@ export default {
     summaryList() {
       return this.$store.getters.getSummaryList
     }
+  },
+  created() {
+    console.log('alo')
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/allocationPayments/Summary/index.vue]: @elsiosanchez remove this created

File: src/components/ADempiere/Form/paymentAllocation/tableFrom.vue [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        this.$refs.allocationPayments.clearSelection()
+      }
+    },
+    tableRowClassName({ row, rowIndex }) {
+      if (row.Difference > 0) {
+        'success-row'
+      }
+      return 'warning-row'
+    },
+    getSummaries(param) {
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/paymentAllocation/tableFrom.vue:112]: Duplicate method, please add support in another utility file, should be called `numberFormat`, to centralize and import it in your implementations.

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**